### ISSUE TITLE
Parse cell line ids from PaxDB metadata

### ISSUE THREAD

**Issue by jonrkarr** [KarrLab/datanator#64]
Ideally, we would convey the id of each cell line to users. Unfortunately, this is hard to parse from PaxDB as they don't represent this consistently.

**Comment by lzy7071** [comment]
Information is missing from most PaxDB data download files. Will add the metadata when the information is made available by PaxDB.

**Comment by lzy7071** [comment]
Inquired about adding the information in the next database release.
